dot/rpc/modules/api.go

```diff
@@ -153,7 +153,8 @@ type SyncStateAPI interface {
 	GenSyncSpec(raw bool) (*genesis.Genesis, error)
 }
 
-//go:generate mockery --name SyncAPI --structname SyncAPI --case underscore --keeptree
+//go:generate mockgen -destination=mock_sync_api_test.go -package $GOPACKAGE . SyncAPI
```

Comment:
nit adding a new line so linters/go tooling doesn't confuse it with the `SyncAPI` comment below.

```suggestion
//go:generate mockgen -destination=mock_sync_api_test.go -package $GOPACKAGE . SyncAPI

```

Reply:
Done in c2e3d69